cluster/gce/gci/configure-helper.sh

```diff
@@ -1932,6 +1932,7 @@ function prepare-konnectivity-server-manifest {
   params+=("--agent-service-account=konnectivity-agent")
   params+=("--kubeconfig=/etc/srv/kubernetes/konnectivity-server/kubeconfig")
   params+=("--authentication-audience=system:konnectivity-server")
+  params+=("--kubeconfig-qps=-1")
```

Comment:
Not loving -1 unlimited. This allows a bug to easily bypass our protections and DOS the KAS. We default to kubeconfig-qps to 5 and kubeconfig-burst to 10. Can we 10x these (50, 100) and see if that's sufficient?

Reply:
I thought about it, as long as this is DaemonSet, we need to scale to node count. What upper limit of node count do we want this to support?

Let's try 100. At 5k cluster bootstrap/scale-up, with agent sync interval 5s, qps spike is 1k, so a given tokenreview shouldn't queue for more than about 10s.